### ISSUE TITLE
[0.76] Error while updating property 'source' of a view managed by: RCTImageView

### ISSUE THREAD

**Issue by 1280103995** [facebook/react-native#47185]
### Description

Today, I saw that RN 0.76 was released, and then I created a new RN project. After running the App, an error was reported.

### Steps to reproduce

1. run ```npx react-native init RN076```
2. Need to install the following packages:
react-native@0.76.0
Ok to proceed? (y) y
3. Need to install the following packages:
@react-native-community/cli@14.1.1
Ok to proceed? (y) y
4. Do you want to install CocoaPods now? Only needed if you run your project in Xcode directly … yes
5. cd "/Users/***/Desktop/RN076" && npx react-native run-android

Later, some Android 15 development dependencies will be downloaded.

### React Native Version

0.76.0

### Affected Platforms

Runtime - Android/iOS

### Output of `npx react-native info`

```text
System:
  OS: macOS 14.4.1
  CPU: (8) arm64 Apple M1 Pro
  Memory: 79.00 MB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 21.6.2
    path: /opt/homebrew/bin/node
  Yarn:
    version: 3.6.4
    path: /opt/homebrew/bin/yarn
  npm:
    version: 10.2.4
    path: /opt/homebrew/bin/npm
  Watchman:
    version: 2024.01.22.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.14.3
    path: /Users/***/.rbenv/shims/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.5
      - iOS 17.5
      - macOS 14.5
      - tvOS 17.5
      - visionOS 1.2
      - watchOS 10.5
  Android SDK: Not Found
IDEs:
  Android Studio: 2023.1 AI-231.9392.1.2311.11255304
  Xcode:
    version: 15.4/15F31d
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 17.0.9
    path: /Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home/bin/javac
  Ruby:
    version: 2.7.6
    path: /Users/***/.rbenv/shims/ruby
npmPackages:
  "@react-native-community/cli":
    installed: 15.0.0-alpha.2
    wanted: 15.0.0-alpha.2
  react:
    installed: 18.3.1
    wanted: 18.3.1
  react-native:
    installed: 0.76.0
    wanted: 0.76.0
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: true
iOS:
  hermesEnabled: true
  newArchEnabled: false
```
```


### Stacktrace or Logs

```text
Error while updating property 'source' of a view managed by: RCTImageView

java.lang.Double cannot be case to com.facebook.react.bridge.ReadableArray
```

### Reproducer

No. I just created a new app normally.

### Screenshots and Videos

<img width="387" alt="211fdb2-c5e6-4962-ae2b-6f39819891e0" src="https://github.com/user-attachments/assets/21fd72e2-3f7f-4a85-a13c-c67afeb6f7bc">
<img width="389" alt="b020ce-d988-4359-b67a-d920aab1def6" src="https://github.com/user-attachments/assets/e7e4d38f-3d47-4a6c-8343-ecb193e6fef5">

**Comment by 1280103995** [comment]
I installed the following dependent libraries in sequence according to the error message:
1. react-native-gesture-handler
2. react-native-device-info
3. @react-native-community/netinfo

But it still reported an error:
<img width="784" alt="39bc-5ca9-4d10-be8e-5d8387a133ef" src="https://github.com/user-attachments/assets/ee7c6d14-9309-4e40-8fc2-8c089645040a">

**Comment by migueldaipre** [comment]
From the error message you are trying to use the react-native-gesture-handler library.

If not, please add a reproducer so we can investigate further.

**Comment by 1280103995** [comment]
> From the error message you are trying to use the react-native-gesture-handler library.
> 
> If not, please add a reproducer so we can investigate further.

In fact, I just created a new project and did not add any dependent libraries. In my comment above, I tried to install some dependent libraries in sequence according to the error message, but it still reported an error.

**Comment by 1280103995** [comment]
> From the error message you are trying to use the react-native-gesture-handler library.
> 
> If not, please add a reproducer so we can investigate further.

ok, I got it. Before I created this new project, I already had a running project, running on port 8081. Then after I created the new project, i ran ```npx react-native run-android```, and it prompted that there was already a 8081 port running, whether to switch to 8082, I chose "yes", and then it ran and reported an error. I need to close all ports, and then run 8082 port alone, and now it works properly.
